documentation/modules/operators/con-application-topic-handling.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='con-application-topic-handling-{context}']
+= Handling changes to topics
+
+[role="_abstract"]
+The Topic Operator is capable of managing changes to topics when there is no single source of truth. 
+The `KafkaTopic` resource and the Kafka topic can undergo independent modifications, where real-time observation of changes may not always be feasible, particularly when the Topic Operator is not operational.
+To handle this, the Topic Operator maintains a topic store that stores relevant information about each topic. 
+It compares the state of the Kafka cluster and Kubernetes with the topic store to determine the necessary changes for synchronization. 
+This evaluation takes place during startup and at regular intervals while the Topic Operator is active.
```

Comment:
Is the topic store used in UTO? In the previous file, you suggest the related environment variables can be removed.

Reply:
Yes, it's not used. I've moved the content around and added a sub-heading to make this section a lot clearer.

> [role="_abstract"]
> How the Topic Operator handles changes to topics depends on the xref:ref-operator-topic-{context}[mode of topic management].
> 
> * For bidirectional topic management, configuration changes are synchronized between the Kafka topic and the `KafkaTopic` resource. Incompatible changes prioritize the Kafka configuration, and the `KafkaTopic` resource is adjusted accordingly.
> * For unidirectional topic management (currently in preview), configuration changes only go in one direction: from the `KafkaTopic` resource to the Kafka topic. Any changes to a Kafka topic managed outside the `KafkaTopic` resource are reverted. 
> 
> == Topic store for bidirectional topic management
> 
> For bidirectional topic management, the Topic Operator is capable of handling changes to topics when there is no single source of truth....